client/js/main.js

```diff
@@ -53,7 +54,31 @@ $(document).ready(function() {
             ribbonModel.set('selectedIndex', 1)
     })
     
+
+    var halfWindowWidth = $(document).width() * 0.5;
+    var optionsFixedWidth = 400;
     var splitPanel  = new SplitPanel({el : "#main-view"})
+    splitPanel.getSection("main-table").initalise({ minWidth: 150, initialWidth: halfWindowWidth < (optionsFixedWidth + SplitPanelSection.sepWidth) ? (optionsFixedWidth + SplitPanelSection.sepWidth) : halfWindowWidth, level: 1});

```

Comment:
is this a bit strange? creating the sections in index.html, and initializing them here? it means that changes always require changes in two places.

what about both creating and initializing them here? (or you could use the 'data-*' attributes to control their initialization in index.html)

what do you think?